core/include/traccc/finding/combinatorial_kalman_filter_algorithm.hpp

```diff
@@ -58,7 +60,8 @@ class combinatorial_kalman_filter_algorithm
     ///
     output_type operator()(
         const default_detector::host& det,
-        const detray::bfield::const_field_t::view_t& field,
+        const detray::bfield::const_field_t<
+            telescope_detector::host::scalar_type>::view_t& field,
```

Comment:
```suggestion
            default_detector::host::scalar_type>::view_t& field,
```

I won't mark the rest. 😉

Reply:
Hope, I got them all now